typescript/src/events/messages/messageUpdateNotify.ts

```diff
@@ -16,38 +15,35 @@ export class UserEvent extends Event {
 	public async run(old: Message, message: GuildMessage) {
 		if (!message.guild || old.content === message.content || message.author.bot) return;
 
-		const [enabled, ignoredChannels, ignoredEdits, ignoredAll, t] = await message.guild.readSettings((settings) => [
-			settings[GuildSettings.Events.MessageEdit],
+		const key = GuildSettings.Channels.Logs[message.channel.nsfw ? 'MessageUpdateNsfw' : 'MessageUpdate'];
+		const [ignoredChannels, channelId, ignoredEdits, ignoredAll, t] = await message.guild.readSettings((settings) => [
```

Comment:
```suggestion
		const [ignoredChannels, logChannelId, ignoredEdits, ignoredAll, t] = await message.guild.readSettings((settings) => [
```
And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.